modules/installation-custom-aws-vpc.adoc

```diff
@@ -67,25 +67,68 @@ The installation program modifies your subnets to add the `kubernetes.io/cluster
 +
 If you prefer to use your own Route 53 hosted private zone, you must associate the existing hosted zone with your VPC prior to installing a cluster. You can define your hosted zone using the `platform.aws.hostedZone` field in the `install-config.yaml` file.
 
-ifndef::aws-china,aws-secret[]
-If you are working in a disconnected environment, you are unable to reach the public IP addresses for EC2 and ELB endpoints. To resolve this, you must create a VPC endpoint and attach it to the subnet that the clusters are using. The endpoints should be named as follows:
+ifndef::aws-secret[]
+If you are working in a disconnected environment, you are unable to reach the public IP addresses for EC2, ELB, and S3 endpoints. Depending on the level to which you want to restrict internet traffic during the installation, the following configuration options are available:
+endif::aws-secret[]
+
+ifdef::aws-secret[]
+A cluster in an SC2S or C2S Region is unable to reach the public IP addresses for the EC2, ELB, and S3 endpoints. Depending on the level to which you want to restrict internet traffic during the installation, the following configuration options are available:
```

Comment:
have very similar wording. I am not sure if that is intentional because I cannot access the preview on my phone for some reason. It does not need to be fixed but I wanted to point it out